app/views/user/_track_progress.erb

```diff
@@ -1,5 +1,5 @@
 <div class="track-info col-xs-4 col-sm-4 col-md-4 col-lg-4">
-  <%= track_icon(track_info.language, '60') %>
+  <%= track_icon(track_info.track.id, '60') %>
```

Comment:
Did track_info always have the `track` attribute? Just double checking since there was no change in the TrackSummary constructor fo track_info